cdist/config.py

```diff
@@ -308,7 +308,8 @@ def onehost(cls, host, host_tags, host_base_path, host_dir_name, args,
                 initial_manifest=args.manifest,
                 add_conf_dirs=args.conf_dir,
                 cache_path_pattern=args.cache_path_pattern,
-                quiet_mode=args.quiet)
+                quiet_mode=args.quiet,
+                exec_path=sys.argv[0])
```

Comment:
Without this one, after changing sys.argv[0] in integration.py in exec/local.py default sys.argv[0] was not seen,
it ended in '' and wrong emulator linking.